### PR TITLE
Fixed random things

### PR DESCRIPTION
**Pull request by dhuangg**
# Conflicts:
#	src/utils.js

### CODE REVIEW

File: src/components/FeatureViews/Accounts/OutOfOffice.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const OutOfOffice = ({instructorID, button}) => {
             const response = await instance.post(
                 "/account/instructor-out-of-office/",
                 {
-                    description,
+                    description: name + " - " + description,
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/OutOfOffice.js]: Use template literals: `${name} - ${description}`

File: src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js [hunk 2]
```diff
@@ -383,21 +382,25 @@ const CourseSessionStatus = () => {
...
-                            <FormGroup>
-                                <FormControlLabel
-                                    control={
-                                        <Switch
-                                            checked={highlightSession}
-                                            onChange={handleHighlightSwitch}
-                                            color="primary"
-                                            value="upcoming-session"
+                        <Grid container alignItems="flex-start">
+                            <Grid item>
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js]: Is it possible to reduce the triple nested `<Grid>`s? (I think the outer two levels can be combined into one tag)

File: src/components/Form/FormUtils.js [hunk 3]
```diff
@@ -75,7 +75,7 @@ export const createDiscountPayload = (form) => {
         case "Payment Method Discount": {
             return {
                 ...discountPayload,
-                "payment_method": form["Discount Rules"]["Payment Method"],
+                "payment_method": form["Discount Rules"]["Payment Method"].toLowerCase(),
```
Comment (NeilAllavarpu) [inline src/components/Form/FormUtils.js]: Is `International Credit Card` option handled properly?

File: src/components/Form/TutoringPriceQuote.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ import {academicLevelParse} from "../../reducers/registrationReducer";
 import InputLabel from "@material-ui/core/InputLabel";
 import {OutlinedSelect} from "../FeatureViews/Scheduler/SchedulerUtils";
 
-const TutoringPriceQuote = ({courseType, handleUpdatePriceFields, tuitionConfirmed, tutoringCategory}) => {
+const TutoringPriceQuote = ({courseType, handleUpdatePriceFields, tuitionConfirmed, tutoringCategory, conditional, form}) => {
```
Comment (NeilAllavarpu) [inline src/components/Form/TutoringPriceQuote.js]: Unused props `conditional` and `form` should be removed.

File: src/utils.js [hunk 5]
```diff
@@ -293,21 +293,21 @@ export const upcomingSession = (sessions, courseID) => sessions.filter((session)
...
 export const tuitionAmount = (courseObject, numSessions) => {
-    let { schedule, hourly_tuition } = courseObject
-    let { end_time, start_date, start_time } = schedule
+    let { schedule, hourly_tuition } = courseObject;
+    let { end_time, start_date, start_time } = schedule;
     // Turns string object into Date string
     let end = `${start_date}${end_time}:00Z`,
         start = `${start_date}${start_time}:00Z`,
-        duration = Math.abs(new Date(end) - new Date(start)) / 36e5
+        duration = Math.abs(new Date(end) - new Date(start)) / 36e5;
```
Comment (NeilAllavarpu) [inline src/utils.js]: Replace magic number `36e5` with a descriptive variable, such as `msPerHour`.

File: src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js (hunk 2 above)
Comment (dhuangg) [reply src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js]: whaaaaoooo didn't know this was a thing for mui3.0 :o

File: src/components/Form/FormUtils.js (hunk 3 above)
Comment (dhuangg) [reply src/components/Form/FormUtils.js]: i think so... but for some reason none of the payment method discounts are being triggered

File: src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js [hunk 6]
```diff
@@ -382,22 +381,24 @@ const CourseSessionStatus = () => {
...
-                            <FormGroup>
-                                <FormControlLabel
-                                    control={
-                                        <Switch
-                                            checked={highlightSession}
-                                            onChange={handleHighlightSwitch}
-                                            color="primary"
-                                            value="upcoming-session"
+                    <Grid item xs={3} container alignItems="flex-start">
+                            <Grid item>
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js]: This (and its contents) is indented one level too many, dedent for better alignment

**Pull request merged**